Source/WebKit/UIProcess/AuxiliaryProcessProxy.cpp

```diff
@@ -328,6 +329,15 @@ void AuxiliaryProcessProxy::didFinishLaunching(ProcessLauncher* launcher, IPC::C
     m_boostedJetsamAssertion = ProcessAssertion::create(*this, "Jetsam Boost"_s, ProcessAssertionType::BoostedJetsam);
 #endif
 
+#if USE(RUNNINGBOARD)
+    m_throttler.didConnectToProcess(*this);
```

Comment:
Maybe it's just me but I'd feel better if we had initialized m_connection before we tell the process throttler that we're connected. You could easily imagine the process throttler trying to do something with the IPC connection and fail because it is not initialized yet. After all, we're claiming we're connected so we should probably have a connection to that process.

Reply:
Ah, yes, these are great points. I will address in the upcoming patch.

Thanks for reviewing!